development/gcp/cloud-run/create-github-issue/create-github-issue.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	port = os.Getenv("LISTEN_PORT")
 	githubOrg = os.Getenv("GITHUB_ORG")
 	githubRepo = os.Getenv("GITHUB_REPO")
+	toolsGithubTokenPath = os.Getenv("TOOLS_SAP_TOKEN_PATH")
```

Comment:
please keep the name consistency:
ENV_NAME => envName
```suggestion
	port = os.Getenv("PORT")
	githubOrg = os.Getenv("GITHUB_ORG")
	githubRepo = os.Getenv("GITHUB_REPO")
	toolsGithubTokenPath = os.Getenv("TOOLS_GITHUB_TOKEN_PATH")
```

Or find different name for the env name. Let's try to keep the names short as possible. Maybe `GHE_TOKEN_PATH`, which translates to "GitHub Enterprise Token Path".